### PR TITLE
Fix HTTPS watermark

### PR DESCRIPTION
**Pull request by jeffkause**
## 🗣 Description ##

Lighten CISA watermark behind the report pages.

## 💭 Motivation and context ##

Fixes dark watermark issue in the HTTPS report Table of Contents.

Screen Shot 2023-11-17 at 12 35 30 PM

## 🧪 Testing ##

Ran test report to verify the CISA watermark is transparent and the words on the pages can be read.

## ✅ Pre-approval checklist ##

- [X] This PR has an informative and human-readable title.
- [X] Changes are limited to a single goal - *eschew scope creep!*
- [X] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [X] All relevant type-of-change labels have been added.
- [X] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [X] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [X] All new and existing tests pass.

## ✅ Post-merge checklist ##

 - [x] Deploy this change to production.

### CODE REVIEW

File: docker-compose.yml
Comment (jsf9k) [inline docker-compose.yml:1]: Please undo the changes to this file.

File: .DS_Store
Comment (jsf9k) [inline .DS_Store:1]: Please remove this file.

File: output/.DS_Store
Comment (jsf9k) [inline output/.DS_Store:1]: Please remove this file.

File: output/artifacts/.DS_Store
Comment (jsf9k) [inline output/artifacts/.DS_Store:1]: Please remove this file.

File: output/artifacts/ocsp-crl.csv
Comment (jsf9k) [inline output/artifacts/ocsp-crl.csv:1]: Please remove this file.

File: output/artifacts/reporting/.DS_Store
Comment (jsf9k) [inline output/artifacts/reporting/.DS_Store:1]: Please remove this file.

File: output/artifacts/reporting/pshtt_reports/cyhy-USCCR-2023-11-17-https-report.pdf
Comment (jsf9k) [inline output/artifacts/reporting/pshtt_reports/cyhy-USCCR-2023-11-17-https-report.pdf:1]: Please remove this file.

File: src/report.sh
Comment (jsf9k) [inline src/report.sh:1]: Please undo the changes to this file.

File: src/secrets/database_creds.yml
Comment (jsf9k) [inline src/secrets/database_creds.yml:1]: Please undo the changes to this file.

**Review by jsf9k — CHANGES_REQUESTED** [review]
You committed a bunch of files that you should not have.

**Comment by jsf9k** [comment]
We will want a version bump for these changes as well.

**Review by jsf9k — APPROVED** [review]
Thanks for the contribution @jeffkause!

**Review by dav3r — APPROVED** [review]
👍 👍

**Comment by jsf9k** [comment]
I updated production to use the Docker image from release 1.2.14 in production.  That is the version that will get used in next weekend's BOD 18-01 scanning run.

**Pull request merged**